user_guide/data_management_access_permissions_basic.rst

```diff
@@ -0,0 +1,112 @@
+  
+- Roles, assigned to the user: for example the sales representatives and system administrators need very different data 
+  and have different access to it. The roles of a user are defined by the system administrator for each user in the 
+  *"Groups and Role"*.
+
+- Ownership type of the entity: when you create a record, you may define its *"Owner"*: these may be the user who can 
+  manage the records or a business unit/organization the members whereof can manage the record. Some record types have 
+  the ownership type *"None"*, which means that all the users can see them. Ownership type of each entity is defined by 
+  the system administrator. 
+
+
+What You Can Be Permitted
+-------------------------
+
+Access to Functionalities
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users with specific roles may be allowed or not allowed to use specific functionality. For example, some users may not 
+have capability to assign tags, to share grids, or to create new users.
+
+These are defined by the system administrator in the *"Capabilities section"* of each role. Only users with a role, for 
+which a capability is enabled, will be able to use the functionality. 
+
+.. note::
+
+    Capabilities of different roles are compounded, i.e. if one role has the right to merge entities, the user will 
+    have this right, regardless of the settings defined for other roles of this user.
+    
+
+Actions with Entities    
+^^^^^^^^^^^^^^^^^^^^^
+
+There is a number of actions that can be permitted for each combination of the role, organization/unit and entity.
+For example, it can be defined that any user with the role "Sales Representative" who belongs to the business unit 
+"California" can create and edit records of Customers, if this records were created by this user or by any other user 
+from this business unit, and can only view (but not edit) similar records created by users from other business units 
+(e.g. "Texas" or "Florida"). At the same time, for users with another role, for example "System Administrator", this 
+records may be even not seen.
+
+The following actions can be permitted for each role, organization/unit and entity:
+  
+  
+- **View**: If, for a specific entity, the action is not available to a user, the user won't see the records 
+  :ref:`Grids <user-guide-ui-components-grids>` nor the :ref:`View pages <user-guide-ui-components-view-pages>` 
+  of this entity records.
+  
+- **Create**: If, for a specific entity, the action is not available to a user, the user won't be able to create new 
+  entity records.
+
+- **Edit**: If, for a specific entity, the action is not available to a user, the user won't be able to edit the entity 
+  records.
+
+- **Delete**: If, for a specific entity, the action is not available to a user, the user won't be able to delete the
+  entity records.
+  
+- **Assign**: If, for a specific entity, the action is not available to a user, the user won't be able to change the owner 
+  of the entity records.
```

Comment:
Edit: **Assign**: If, for a specific entity, this action is not available to a user, the user won't be able to change the owner of the entity records.